src/processors/data-processor.js

```diff
@@ -272,7 +272,13 @@ export function getFieldsFromData(data, fieldOrder) {
   // add a check for epoch timestamp
   const metadata = Analyzer.computeColMeta(
     data,
-    [{regex: /.*geojson|all_points/g, dataType: 'GEOMETRY'}],
+    [
+      {regex: /.*geojson|all_points/g, dataType: 'GEOMETRY'},
+
+      // s2 token
+      {regex: /token/g, dataType: 'STRING'},
```

Comment:
https://github.com/keplergl/kepler.gl/pull/891 just landed
please rebase and check, you might  not need these regexes